notify/config.py

```diff
@@ -69,23 +70,13 @@
         "notify_backends": {
             "type": "object",
             "properties": {
-                "*": {
-                    "type": "object",
-                    "properties": {
-                        "salesforce": {
-                            "type": "object",
-                            "properties": {
-                                "atuh_url": {"type": "string"},
-                                "username": {"type": "string"},
-                                "password": {"type": "string"},
-                                "environment": {"type": "string"},
-                                "client_id":  {"type": "string"},
-                                "client_secret": {"type": "string"},
-                                "organization_id": {"type": "string"}
-                            }
-                        }
-                    }
-                }
+                "*": {"type": "object"}
+            }
+        },
+        "logging": {
```

Comment:
I would suggest not to add this functionality right now and do it properly later by dying [the appropriate configuration of logging throught](https://docs.python.org/3/library/logging.config.html) `fileConfig` or `dictConfig` in `oss-lib`.